assembl/views/__init__.py

```diff
@@ -659,6 +660,24 @@ def redirector(request):
         'home', discussion_slug=request.matchdict.get('discussion_slug')))
 
 
+def sanitize_next_view(next_view):
+    if next_view and ':/' in next_view:
```

Comment:
The `:/` check is not necessary. A `urlparse` will yield no netloc if it doesn't exist.

Reply:
true, but that way I get a fast path for most common case. urlparse is not nothing.